Terminal.Gui/ConsoleDrivers/CursesDriver/CursesDriver.cs

```diff
@@ -369,7 +369,7 @@ static Key MapCursesKey (int cursesKey)
 		case Curses.KeyDeleteChar: return Key.DeleteChar;
 		case Curses.KeyInsertChar: return Key.InsertChar;
 		case Curses.KeyTab: return Key.Tab;
-		case Curses.KeyBackTab: return Key.BackTab;
+		case Curses.KeyBackTab: return Key.Tab | Key.ShiftMask;
 		case Curses.KeyBackspace: return Key.Backspace;
```

Comment:
Test if that really work on Linux, because normally it doesn't recognize the `Shift` key.